.github/workflows/jacktrip.yml

```diff
@@ -462,9 +450,6 @@ jobs:
           if [[ "${{ matrix.weakjack }}" == true ]]; then 
             CONFIG_STRING="-Dweakjack=true $CONFIG_STRING"
           fi
-          if [[ "${{ runner.os }}" == 'Linux' ]]; then
-            CONFIG_STRING="-Dwerror=true $CONFIG_STRING"
-          fi
```

Comment:
I added this so we don't regress with code quality (at least on Linux). What warning is the reason for the removal?

Reply:
RtAudio generates warnings when the ALSA backend is enabled. This caused the builds to fail.

Reply:
Perhaps there is a way to do this without it also applying to subprojects?

Reply:
`-Drtaudio:werror=false` should work in our case I think.

Reply:
Cool thanks, I'll give it a try!

Reply:
An alternative would be to add `include_type: 'system'` to the rtaudio dependency, like we did with Qt, but it has additional implications like ordering of build flags.

Reply:
Nice, looks like rtaudio:werror worked. :)

Reply:
Looks like it worked!